src/lib/seam/use-seam-client.ts

```diff
@@ -44,15 +45,17 @@ export function useSeamClient(): {
   return { client: data ?? null, isLoading, isError, error }
 }
 
-/**
- * You'll almost always want to supply a user identifier key, but if you don't
- * we'll automatically pick a fingerprint for this user. The user interface
- * will show warnings when you use a fingerprint because this client session
- * is bound to this machine, and is effectively ephemeral.
- */
```

Comment:
Jsdocs also appear in type hints for consumers, not everyone is going to look at the warning panel

Reply:
Unrelated but github never committed your suggestion for fingerprintValue which is concerning and annoying

Reply:
Ah no worries about the fingerprintValue, we will resolve those open comments as we go.

I didn't know that about the comments showing up in the type hints. Can we have both? For the type hints though, I think the comment would need to be on something directly used by the consumer, and right now we don't have any jsdoc comments for the public API. If we did we could annotate the `userIdentifierKey` prop with the warning.